package/pom.xml

```diff
@@ -47,16 +47,48 @@
                 <version>${maven-assembly-plugin.version}</version>
                 <executions>
                     <execution>
-                        <id>distribution-package</id>
+                        <id>full-package</id>
                         <phase>package</phase>
                         <goals>
                             <goal>single</goal>
                         </goals>
                         <configuration>
                             <appendAssemblyId>false</appendAssemblyId>
-                            <finalName>arcadedb-${project.version}</finalName>
+                            <finalName>arcadedb-package-${project.version}</finalName>
```

Comment:
What about using `arcadedb-full` as a name? So everybody understand it's the full version?

Reply:
was my first choice. But then I preferred to not create problems to people that is referring, maybe in their scripts, to *arcadedb-package*

Reply:
I think it's the opposite: now arcadedb-package is just a sub-project to pack up the distribution and everybody is using just "arcadedb-<version>".

Reply:
no, it's deployed on maven central: https://repo.maven.apache.org/maven2/com/arcadedb/arcadedb-package/23.9.1/

Reply:
I never noticed we published the whole distribution on Maven, not only GitHub! But nobody is linking that package because it's a zip. not a jar. So we can even get rid of that and nobody would notice. The zip file is directly available on github.

The only doubt is about the name of the full distribution. In Docker we keep the current name (no suffix), that doesn't break compatibility 👍 But for the release we change it to "package" just to stick with the current zip deployed on maven, but nobody I think ever used it.

What if we remove the suffix from it? So we'd have (I usually upload the `.tar.gz` instead of `.zip` on github because it keeps the UNIX permissions on the .sh files):

- arcadedb-VERSION.tar.gz: as before, the full-fledged package with all the plugins inside
- arcadedb-minimal-VERSION.tar.gz
- arcadedb-headless-VERSION.tar.gz

And same for docker:

- arcadedata/arcadedb: as before, the full version (non rename)
- arcadedata/arcadedb-minimal
- arcadedata/arcadedb-headless

Reply:
Ok, this commit https://github.com/ArcadeData/arcadedb/pull/1280/commits/44216465178eae410407927861ac33d9666233ea

does that